tests/helm/testdata/goldenfile/metrics_collector_otc/autoscaling.input.yaml

```diff
@@ -0,0 +1,12 @@
+sumologic:
+  metrics:
+    enabled: true
+    collector:
+      otelcol:
+        enabled: true
+        autoscaling:
+          enabled: true
+          minReplicas: 15
+          maxReplicas: 30
+          targetCPUUtilizationPercentage: 95
+          targetMemoryUtilizationPercentage: 90
```

Comment:
Can we rename this file to `custom.input.yaml` and use it for other settings as well? Alternatively, if you want a test for just autoscaling, you can write it in Go.

Reply:
Yes, we can :)